packages/react-core/src/components/LabelGroup/LabelGroup.tsx

```diff
@@ -58,6 +58,8 @@ export interface LabelGroupProps extends React.HTMLProps<HTMLUListElement> {
   hasEditableTextArea?: boolean;
   /** @beta Additional props passed to the editable textarea. */
   editableTextAreaProps?: any;
+  /** @beta Control for adding new labels */
+  addLabel?: React.ReactNode;
```

Comment:
Maybe `addLabelControl` or `addLabelAction` for a name would be more descriptive of prop intent?